### PR TITLE
rtabmap: unstable-2022-02-07 -> unstable-2022-09-24

### PR DESCRIPTION
**Pull request by superherointj**
rtabmap: unstable-2022-02-07 -> unstable-2022-09-24

Fixes error:

* linux-x86_64:
  image
  logs: https://termbin.com/ui5e

* linux-aarch64:
  image
  logs: https://termbin.com/90g0

Maintainers: @ckie

### CODE REVIEW

File: pkgs/applications/video/rtabmap/default.nix [hunk 1]
```diff
@@ -55,5 +55,6 @@ stdenv.mkDerivation rec {
     license = licenses.bsd3;
     maintainers = with maintainers; [ ckie ];
     platforms = with platforms; linux;
+    broken = true; # at 2022-09-23
```
Comment (SuperSandro2000) [inline pkgs/applications/video/rtabmap/default.nix:58]: Please add a short comment why it got marked broken.

Comment (superherointj) [reply pkgs/applications/video/rtabmap/default.nix:58]: Done

**Comment by ckiee** [comment]
It's already [fixed upstream](https://github.com/introlab/rtabmap/commit/cf64b20e1fd5b7aca6f5f708e9b790aa41e799db), update or patch?

**Comment by superherointj** [comment]
> It's already [fixed upstream](https://github.com/introlab/rtabmap/commit/cf64b20e1fd5b7aca6f5f708e9b790aa41e799db), update or patch?

Updated. Please review.

**Review by ckiee — APPROVED** [review]
A few new probably `dlopen`-ed runtime dependencies added, builds and starts up. Didn't test with any actual hardware.

**Comment by superherointj** [comment]
Thanks ckiee.

**Pull request merged**